package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "run-vcpkg-action",
+  "version": "11.0.0",
+  "description": "GitHub action for running vcpkg with Binary Caching stored on the GitHub Action cache.",
+  "repository": {
+    "type": "git",
+    "url": " https://github.com/step-security/run-vcpkg-action"
+  },
+  "author": "Step Security (https://github.com/lukka)",
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf ./build",
+    "deployAssets": "cp ./node_modules/@lukka/assets-lib/src/matchers/*.json ./dist/",
+    "build": "npm run deployAssets && npx tsc",
+    "lint": "npx eslint .",
+    "test": "npx jest",
+    "pack": "npx ncc build build/src/action.js -o dist",
```

Comment:
rename `pack` script to `build` & `build` script to `pre-build`
Currently `pack` script is used to build dist folder and for our pre-release step we prefer it to be as `build`
Also rebuild `dist` folder after renaming!

Reply:
done

Reply:
done